app/javascript/components/applications/ApplicationsTable.js

```diff
@@ -17,7 +18,9 @@ const SubComponent = (row) => (
 /*      Presenter                         */
 /******************************************/
 
-const ApplicationsTable = ({ applications, onFetchData, loading, pages, rowsPerPage }) => {
+const ApplicationsTable = ({ applications, onFetchData, loading, pages, rowsPerPage, maxPages }) => {
+  const maxPagesMsg = `You have reached the maximum number of records we can display at this time. There are ${pages * rowsPerPage} records that match your selected filters, but the maximum we can display is ${MAX_SERVER_LIMIT}. Please use filters to narrow the number of matching records.`
```

Comment:
Would it be easy to format the numbers with commas?

Reply:
also, is pages here the rounded up number, or the number of pages with the decimal? If it's the rounded number, would it be possible to have the real number here? There could be a partial page at the end and we don't want to display incorrect information.